config/jobs/kubernetes/sig-node/k8s-release-branches.yaml

```diff
@@ -1,5 +1,6 @@
 periodics:
 - name: ci-kubernetes-node-release-branch-1-25
+  cluster: k8s-infra-prow-build
```

Comment:
You maybe have to remove the project type as that is not defined in the new cluster.

Reply:
Updated.